.clang-format

```diff
@@ -0,0 +1,28 @@
+BasedOnStyle: LLVM
+IndentWidth: 8
```

Comment:
This is also one thing that I am not sure if we are all on the same page. I use tab width 4 in my editor, but I am fine to adjust on 8 if we will be on the same page here.